frontend/src/components/MeetingRoom/VideoTileCanvas/VideoTileCanvas.tsx

```diff
@@ -14,6 +15,7 @@ import getLayoutBoxes from '../../../utils/helpers/getLayoutBoxes';
 import useActiveSpeaker from '../../../hooks/useActiveSpeaker';
 
 export type VideoTileCanvasProps = {
+  connected: boolean | null;
```

Comment:
Instead of passing as props, could we grab `connected` from the session context in the VideoTileCanvas component instead?

Reply:
sounds good